test/integration/ddl_sharding_key_test.lua

```diff
@@ -0,0 +1,182 @@
+local fio = require('fio')
```

Comment:
There is no test that check an error "Specify bucket_id" in case if sharding is performed by fields that are not a part of primary index.

Reply:
still relevant

Reply:
added three tests:
```
    ddl_sharding_key[vinyl].test_incomplete_sharding_key_delete ... (0.012s) Ok
    ddl_sharding_key[vinyl].test_incomplete_sharding_key_get ... (0.008s) Ok   
    ddl_sharding_key[vinyl].test_incomplete_sharding_key_update ... (0.008s) Ok                                                                               
```